deploy/bin/configure.js

```diff
@@ -42,10 +42,6 @@ function* configure() {
     currentConfig[key] = (yield prompt(humanDesc)) || defaultValue;
   }
 
-  console.log();
-  console.log(util.inspect(currentConfig, { colors: true }));
-  console.log();
-
   // Yeah bad things will happen if rejected too often...
   if (!(yield prompt.confirm("Does this look right? "))) {
```

Comment:
We removed outputting the contents of the config to confirm if it looks correct, do we want to remove this prompt as well?